cpu/esp32/doc.txt

```diff
@@ -467,24 +528,24 @@ When the `esp_log_tagged` module is used, all log messages are tagged with
 additional information: the type of message, the system time in ms, and the
 module or function in which the log message is generated. For example:
 
-```
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 I (663) [main_trampoline] main(): This is RIOT! (Version: 2019.10-devel-437-gf506a)
-```
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Either the `LOG_*` macros as defined in
 [system logging header](http://doc.riot-os.org/log_8h.html) or the tagged
```

Comment:
```diff
- [system logging header](http://doc.riot-os.org/log_8h.html)
+ [system logging header](https://doc.riot-os.org/core_2include_2log_8h.html)
```

Reply:
I added an anchor to the doc there and reference this instead, as future Doxygen versions might produce slightly different links. The anchor should hopefully remain stable.